kumascript/macros/ListSubpagesForSidebar.ejs

```diff
@@ -34,111 +30,90 @@ const overview = mdn.localString({
+    pages.unshift({
+        ...parent,
+        title: overview
+    });
+}
+
+let output = '';
+
+const containsTag = page.hasTag;
+const htmlEscape = kuma.htmlEscape;
+const openTag = '<ol>';
+const closeTag = '</ol>';
+const code = !$1 ? '<code>' : '';
+const endcode = !$1 ? '</code>' : '';
+
+const badges = {
+    ExperimentalBadge: await template("ExperimentalBadge"),
+    NonStandardBadge: await template("NonStandardBadge"),
+    DeprecatedBadge: await template("DeprecatedBadge"),
+    ObsoleteBadge: await template("ObsoleteBadge"),
 }
 
 // Trims the title, returning only the text
 // between the start and end delimiter characters.
 // Does nothing if both are null or empty.
 function trimTitle(title) {
-    var startIndex;
-    var endIndex;
-
-    startIndex = 0;
-    if (startDelim) {
-        startIndex = title.indexOf(startDelim);
-        if (startIndex) {
-            startIndex += 1;
-        }
-    }
+    const startIndex = startDelim ? Math.max(title.indexOf(startDelim), 0) : 0;
 
-    endIndex = 0;
-    if (endDelim) {
-        endIndex = title.indexOf(endDelim)+1;
-    }
-    if (!endIndex) {
-        endIndex = title.length;
-    }
+    const endIndex = (endDelim ? title.indexOf(endDelim) + 1 : title.length) ||
+        title.length; // if endIndex is 0 (not found), then use title.length
 
     return title.substring(startIndex, endIndex);
 }
 
 function createLink(item) {
-    var result = '';
-    if (!item.url) {
-        return '';
-    }
+    const url = item.url.replace('en-US', locale);
+    let title = htmlEscape(item.title);
 
-    var url = item.url.replace('en-US', locale);
-    var title = htmlEscape(trimTitle(item.title));
-
-    if (locale != 'en-US') {
-        item.translations.forEach(function(translation){
-            if(translation.locale === locale) {
-                url = translation.url;
+    if (locale !== 'en-US') {
+        for (const translation of item.translations) {
+            if (translation.locale === locale) {
                 title = htmlEscape(trimTitle(translation.title));
             }
-        });
+        }
     }
 
-    result += '<li>';
+    const pageBadges = [];
 
     if (containsTag(item, 'Experimental')) {
-        result += badges.ExperimentalBadge;
+        pageBadges.push(badges.ExperimentalBadge);
     }
 
     if (containsTag(item, 'Non-standard') || containsTag(item, 'Non Standard')) {
-        result += badges.NonStandardBadge;
+        pageBadges.push(badges.NonStandardBadge);
     }
 
     if (containsTag(item, 'Deprecated')) {
-        result += badges.DeprecatedBadge;
+        pageBadges.push(badges.DeprecatedBadge);
     }
 
     if (containsTag(item, 'Obsolete')) {
-        result += badges.ObsoleteBadge;
+        pageBadges.push(badges.ObsoleteBadge);
     }
 
-    result += '<a href="' + url + '">' + code + title + endcode + '</a></li>';
+    const result = `<li>${pageBadges.join('')}<a href="${url}">${code}${title}${endcode}</a></li>`
```

Comment:
Once we have a more descriptive `wrapInCode` variable, we can replace variables `code` and `endcode` like this:

```suggestion
    let linkContent = title;
    if (wrapInCode) {
      linkContent = `<code>${linkContent}</code>`;
    }
    
    const result = `<li>${pageBadges.join('')}<a href="${url}">${linkContent}</a></li>`
```